### ISSUE TITLE
[Priest] Mind Soothe has no effect

### ISSUE THREAD

**Issue by Illiyara** [dalaranwow/dalaran-wow#4199]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**:
Casting Mind Soothe on a humanoid mob has no effect on its aggro range. I believe this broke when the way mobs aggro and reset was fixed (so you could kite a mob a long distance). It used to work before.

**Expected behaviour**:
Mobs with the mind soothe debuff should have a reduced aggro range.

**Steps to reproduce the problem**:
1. Approach a stationary humanoid mob slowly until you take aggro. Note the distance required. 
2.  Deaggro the mob.
3.  Cast mind soothe on the mob.
4. Repeat 1 and notice there's no difference.

**Include proofs for this behaviour**
You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
http://wow.gamepedia.com/Mind_Soothe
http://wotlk.openwow.com/spell=453

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
without spell aggro range:

![GitHub Logo](http://i.imgur.com/fd5lwTK.jpg)

with spell cast aggro range

![GitHub Logo](http://i.imgur.com/u9lQWA8.jpg)

so it works fine on PTR. The fix will be applied on live with further merges.
